Java/libraries/recognizers-text-datatypes-timex-expression/src/test/java/com/microsoft/recognizers/text/datatypes/timex/expression/TestTimexRangeResolve.java

```diff
@@ -0,0 +1,1099 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.microsoft.recognizers.text.datatypes.timex.expression;
+
+import com.microsoft.recognizers.datatypes.timex.expression.TimexCreator;
+import com.microsoft.recognizers.datatypes.timex.expression.TimexProperty;
+import com.microsoft.recognizers.datatypes.timex.expression.TimexRangeResolver;
+
+import java.math.BigDecimal;
+import java.util.ArrayList;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+public class TestTimexRangeResolve {
```

Comment:
Minor: Should be "TestTimexRangeResolver".

Reply:
Done!